core/src/main/res/values/strings.xml

```diff
@@ -170,24 +170,24 @@
     <string name="share_file_label">Share File</string>
     <string name="share_website_url_label">Website address</string>
     <string name="share_feed_url_label">Podcast feed URL</string>
-    <string name="feed_delete_confirmation_msg">Please confirm that you want to delete the podcast \"%1$s\" and ALL its episodes (including downloaded episodes).</string>
-    <string name="feed_delete_confirmation_local_msg">Please confirm that you want to remove the podcast \"%1$s\". The files in the local source folder will not be deleted.</string>
-    <string name="feed_remover_msg">Removing podcast</string>
+    <string name="feed_delete_confirmation_msg">Please confirm deletion of the \"%1$s\" podcast and ALL its episodes (including downloaded ones).</string>
+    <string name="feed_delete_confirmation_local_msg">Please confirm removal of the \"%1$s\" podcast. Files in the local source folder will not be deleted.</string>
+    <string name="feed_remover_msg">Removing podcast…</string>
     <string name="load_complete_feed">Refresh complete podcast</string>
     <string name="multi_select">Multi select</string>
     <string name="select_all_above">Select all above</string>
     <string name="select_all_below">Select all below</string>
     <string name="filtered_label">Filtered</string>
-    <string name="refresh_failed_msg">{fa-exclamation-circle} Last Refresh failed. Tap to view details.</string>
+    <string name="refresh_failed_msg">{fa-exclamation-circle} Last refresh failed. Tap to view details.</string>
     <string name="open_podcast">Open Podcast</string>
-    <string name="please_wait_for_data">Please wait until the data is loaded</string>
-    <string name="updates_disabled_label">Updates disabled</string>
+    <string name="please_wait_for_data">Please wait until the data is loaded…</string>
```

Comment:
'Please wait' isn't really an ongoing action ('downloading…' would be). Please revert (or we have to find another way to phrase it - but I'm not sure where in the UI this is shown).

Reply:
So the data isn't being loaded when the string is shown? I don't understand how that works.